arbitrum-docs/for-devs/concepts/differences-between-arbitrum-ethereum/solidity-support.mdx

```diff
@@ -0,0 +1,26 @@
+| `block.difficulty` | Returns the constant 1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| `block.prevrandao` | Returns the constant 1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| `block.number`     | Returns an "estimate" of the L1 block number at which the sequencer received the transaction. For more information, see [Block numbers and time](/for-devs/concepts/differences-between-arbitrum-ethereum/block-numbers-and-time.mdx).                                                                                                                                                                                                                                                                                                                                                            |
+| `msg.sender`       | Works the same way it does on Ethereum for regular L2 to L2 transactions. For L1 to L2 transactions (also known as "retryable tickets"), it will return the L2 address alias of the L1 contract that triggered the message. For more information, see [retryable ticket address aliasing](/arbos/l1-to-l2-messaging.mdx#address-aliasing).                                                                                                                                                                                                                                                        |
```

Comment:
```suggestion
| `msg.sender`       | Works the same way it does on Ethereum for regular L2 to L2 transactions. For transactions submitted via the delayed inbox, it will return the L2 address alias of the L1 contract that triggered the message. For more information, see [address aliasing](/arbos/l1-to-l2-messaging.mdx#address-aliasing).                                                                                                                                                                                                                                                        |
```